libsrc/vic20/tgi/vic20-hi.s

```diff
@@ -250,23 +251,21 @@ PATTERN_SOLID:
 
 ; Initialize variables
 
-        ldx     #$FF
-        stx     BITMASK
+        ldy     #$FF
+        sty     BITMASK
 
 ; Make screen columns.
 
-        lda     #<SBASE
-        sta     tmp2
         lda     #>SBASE
         sta     tmp2+1
-        inx                     ; (ldx #$00)
-        stx     ERROR           ; Set to TGI_ERR_OK
+        iny                     ; (ldy #$00)
+        sty     tmp2
+        sty     ERROR           ; Set to TGI_ERR_OK
```

Comment:
Style nitpick: I prefer that the
```asm
iny
sty ERROR
```
be placed above the comment "Make screen columns."  And, the `sty tmp2` should be put where the `sta tmp2` was.

Reply:
Fixed

Reply:
It really is the `sty ERROR` that needs to be moved, it isn't a part of the "Make screen columns" section.

Reply:
Ah, right.